DoePaAdminDataModel/Stammdaten/Auftrag.cs

```diff
@@ -21,6 +21,8 @@ public class Auftrag
         public List<Auftragsposition> Auftragspositionen { get; set; }
 
         public Mitarbeiter VerantwortlicherMitarbeiter { get; set; }
+        
+        public Geschaeftsjahr ZugehoerigesGeschaeftsjahr { get; set; }
```

Comment:
@timo2o1o Haha oh sorry, I'm still used to naming the attributes directly without further descriptive naming like here.... I had stupidly overlooked your explicit naming in the issue... :(

Reply:
> @timo2o1o Haha oh sorry, I'm still used to naming the attributes directly without further descriptive naming like here.... I had stupidly overlooked your explicit naming in the issue... :(

All good. We can totally discuss the naming conventions if we establish our coding guidelines. This is just the way I like it, That does not necessarily mean, that we have to do it this way :).